donkeycar/parts/datastore_v2.py

```diff
@@ -46,6 +50,9 @@ def __enter__(self):
         return self
 
     def writeline(self, contents):
+        if self.method == 'r':
+            raise Exception(f'Seekable {self.file} is read-only.')
```

Comment:
Only minor comment - I try to avoid throwing base `Exceptions` but rather use more specific ones, here probably `RuntimeError`

Reply:
Done.